tests/test_project.py

```diff
@@ -98,7 +98,7 @@ def test_repr(self):
         assert p == self.project
 
     def test_str(self):
-        str(self.project) == self.project.id
+        str(self.project) == f"Project.get_project({self.project.root_directory()})"
```

Comment:
Well it would help if this test actually contained an assertion... 😂 

Looks like this test has remained unchanged since https://github.com/glotzerlab/signac/commit/bbf7e7b0db9164bf10efaa84ab2888a8bd15e307 way back in 2016. Probably good to enable it now.

Reply:
> To know oneself, one should assert oneself.
> -- Albert Camus